masu/theme.json

```diff
@@ -0,0 +1,699 @@
+{
+	"version": 2,
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"palette": [
+				{
+					"color": "#501511",
```

Comment:
Was there a decision made somewhere about the color naming convention going forward?

Reply:
I was going to add the same. I wonder why colors have custom slugs against primary/secondary etc.

Reply:
I don't know if there is a decision has been made. As we saw in the thread about the convention used in Rainfall, and I wanted to try it. But I think I'll return to the current convention as many patterns could look awful in this theme.